python/ray/_private/worker.py

```diff
@@ -2587,6 +2588,9 @@ def _mode(worker=global_worker):
 
 
 def _make_remote(function_or_class, options):
+    if not function_or_class.__module__:
+        function_or_class.__module__ = "global"
```

Comment:
This is a bit sketch -- it is fixing https://github.com/ray-project/ray/issues/4758 but most likely there is a better fix :)